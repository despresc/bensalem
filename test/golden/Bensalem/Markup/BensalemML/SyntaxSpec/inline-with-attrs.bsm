\inlineelt[a1={thing} a2=[a3={x}]]
