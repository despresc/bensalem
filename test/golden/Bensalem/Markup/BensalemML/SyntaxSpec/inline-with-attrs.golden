fromList
 [ ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 0
     , srcLine = 1
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 10
     , srcLine = 1
     , srcCol = 11
     }
    }
   , elementName = "inlineelt"
   , elementAttrs = fromList
    [ Attr
     ( SrcSpan
      { srcSpanName = ""
      , srcSpanStart = SrcPos
       { srcOffset = 11
       , srcLine = 1
       , srcCol = 12
       }
      , srcSpanEnd = SrcPos
       { srcOffset = 13
       , srcLine = 1
       , srcCol = 14
       }
      }
     ) "a1"
     ( AttrValMarkup
      ( fromList
       [ PlainText "thing" ]
      )
     )
    , Attr
     ( SrcSpan
      { srcSpanName = ""
      , srcSpanStart = SrcPos
       { srcOffset = 22
       , srcLine = 1
       , srcCol = 23
       }
      , srcSpanEnd = SrcPos
       { srcOffset = 24
       , srcLine = 1
       , srcCol = 25
       }
      }
     ) "a2"
     ( AttrValSet
      ( fromList
       [ Attr
        ( SrcSpan
         { srcSpanName = ""
         , srcSpanStart = SrcPos
          { srcOffset = 26
          , srcLine = 1
          , srcCol = 27
          }
         , srcSpanEnd = SrcPos
          { srcOffset = 28
          , srcLine = 1
          , srcCol = 29
          }
         }
        ) "a3"
        ( AttrValMarkup
         ( fromList
          [ PlainText "x" ]
         )
        )
       ]
      )
     )
    ]
   , elementArg = fromList []
   }
  )
 , LineEnd
 ]